core/templates/pages/exploration-editor-page/exploration-save-and-publish-buttons/exploration-save-and-publish-buttons.component.ts

```diff
@@ -129,13 +140,24 @@ angular.module('oppia').component('explorationSaveAndPublishButtons', {
         $scope.saveIsInProcess = false;
         $scope.publishIsInProcess = false;
         $scope.loadingDotsAreShown = false;
-
```

Comment:
Undo unwanted change* (i.e, don't remove the newline.)

Reply:
DONE